modular_skyrat/modules/goofsec/code/sol_fed.dm

```diff
@@ -215,12 +548,186 @@ GLOBAL_LIST_INIT(emergency_responders, list())
 					cop.key = chosen_candidate.key
 
 					//Give antag datum
-					var/datum/antagonist/ert/request_911/ert_antag = new /datum/antagonist/ert/request_911/condom_destroyer
+					var/datum/antagonist/ert/request_911/ert_antag = new type_to_summon
 
 					cop.mind.add_antag_datum(ert_antag)
 					cop.mind.set_assigned_role(SSjob.GetJobType(ert_antag.ert_job_path))
 					SSjob.SendToLateJoin(cop)
+					cop.grant_language(/datum/language/common, TRUE, TRUE, LANGUAGE_MIND)
+
+					var/obj/item/gangster_cellphone/phone = new() // biggest gang in the city
+					phone.gang_id = cell_phone_number
+					phone.name = "[cell_phone_number] branded cell phone"
+					var/phone_equipped = phone.equip_to_best_slot(cop)
+					if(!phone_equipped)
+						to_chat(cop, "Your [phone.name] has been placed at your feet.")
+						phone.forceMove(get_turf(cop))
 
 					//Logging and cleanup
 					log_game("[key_name(cop)] has been selected as an [ert_antag.name]")
-					numagents--
+					agents_number--
+
+/obj/item/solfed_reporter/swat_caller
+	name = "S.W.A.T. Backup Caller"
+	desc = "Use this in-hand to vote to call SolFed S.W.A.T. backup. If your entire team votes for it, SWAT will be dispatched."
+	type_to_check = /datum/antagonist/ert/request_911
+	type_of_callers = "911_responders"
+	announcement_source = "Sol Federation S.W.A.T."
+	fine_station = TRUE
+	ghost_poll_msg = "The Sol-Fed 911 services have requested a S.W.A.T. backup. Do you wish to become a S.W.A.T. member?"
+	amount_to_summon = 6
+	type_to_summon = /datum/antagonist/ert/request_911/condom_destroyer
+	summoned_type = "swat"
+	announcement_message = "Hello, crewmembers. Our emergency services have requested S.W.A.T. backup, either for assistance doing their job due to crew \
+		impediment, or due to a fraudulent 911 call. We have billed the station $20,000 for this, to cover the expenses of flying a second emergency response to \
+		your station. Please comply with all requests by said S.W.A.T. members."
```

Comment:
You've duplicated that message, it's the exact same as the other one, it seems.

Reply:
Fixed